admin/partials/engagement-cloud-admin-display.php

```diff
@@ -74,7 +74,7 @@ public function display_plugin_setup_page() {
 			ABSPATH;
 
 		global $wpdb;
-		$table_name = $wpdb->prefix . $this->plugin_name;
+		$table_name = $wpdb->prefix . "dotmailer_email_marketing";
```

Comment:
Can we create a global variable to hold "dotmailer_email_marketing" and then reference that variable everywhere else?

Reply:
I have no idea. I guess, we can, I can look it into.

Reply:
Also, perhaps that shouldn't be in this PR as it's beyond its purpose?